k8s-do/k8s/ingress-tls/Chart.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: v2
+name: ingress-tls
+description: ingress-tls installs and configures cert-manager, an ingress controller and a Let's Encrypt Issuer for automating TLS Certificates on AsyncAPI K8s services.
+type: application
+version: 0.1.0
+appVersion: "v1.6.1" # version of cert-manager
```

Comment:
Should it be with `v` prefix?

Reply:
It doesn't matter, this has nothing to do with the chart version. But the version of cert-manager is with v prefix https://github.com/jetstack/cert-manager/releases/tag/v1.6.1

Reply:
Ok, thanks for explanation!